### ISSUE TITLE
Trailing comma => PHP Parse error on PHP 7.2

### ISSUE THREAD

**Issue by leoloso** [10up/plugin-check#265]
### Describe the bug

When using PHP 7.2 and checking a plugin, an error happens:

```
PHP Parse error:  syntax error, unexpected ')' in /app/wordpress/wp-content/plugins/plugin-check/checks/plugin-updaters.php on line 12
```

The code is this one, with a trailing comma:

```php
return new Error(
  'plugin_updater_detected',
  'Plugin Updater detected. Use of the Update URI header is not helpful in plugins hosted on WordPress.org.',
);
```

And [trailing commas are supported from PHP 7.3](https://php.watch/versions/7.3/func-call-trailing-comma)

### Steps to Reproduce

1. Use PHP 7.2
2. Check a plugin
3. Error

### Environment information

- PHP version: 7.2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by leoloso** [comment]
Using PHP 7.3 it works well

**Comment by bordoni** [comment]
Fixed on `0.2.1`
